Error when trying to compile - missing install-deps.sh support for Debian 11 Linux

### Contour Terminal version

master, at 6e047fd

### Installer source

Github: source code cloned

### Operating System

debian 11 (bullseye)

### Architecture

x86-64

### Other Software

_No response_

### Steps to reproduce

1. Clone repo and cd into it.
2. Follow source compilation instructions:
3. run `./scripts/install-deps.sh`
4. run `mkdir build`
5. run `cmake -S . -B build`
6. run `cmake --build build/`


### Expected Behavior

Expected the program to be compiled.

### Actual Behavior

Got error message:
```
[  1%] Building CXX object _deps/sources/libunicode-44969c6d80e44a4731b584d047ba108769926835/src/unicode/CMakeFiles/unicode.dir/emoji_segmenter.cpp.o
In file included from /home/myusername/contour/_deps/sources/libunicode-44969c6d80e44a4731b584d047ba108769926835/src/unicode/emoji_segmenter.cpp:15:
/home/myusername/contour/_deps/sources/libunicode-44969c6d80e44a4731b584d047ba108769926835/src/unicode/emoji_segmenter.h:113:87: error: expected class-name before ‘{’ token
  113 | t::formatter<unicode::PresentationStyle>: fmt::ostream_formatter {};
      |                                                                  ^

gmake[2]: *** [_deps/sources/libunicode-44969c6d80e44a4731b584d047ba108769926835/src/unicode/CMakeFiles/unicode.dir/build.make:108: _deps/sources/libunicode-44969c6d80e44a4731b584d047ba108769926835/src/unicode/CMakeFiles/unicode.dir/emoji_segmenter.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:398: _deps/sources/libunicode-44969c6d80e44a4731b584d047ba108769926835/src/unicode/CMakeFiles/unicode.dir/all] Error 2
gmake: *** [Makefile:171: all] Error 2
```

### Additional notes

_No response_

--- a/scripts/install-deps.sh
+++ b/scripts/install-deps.sh
@@ -432,8 +432,12 @@ main()
         pop)
             install_deps_popos
             ;;
-        ubuntu|neon|debian)
+        ubuntu|neon)
             install_deps_ubuntu
+            ;;
+        debian)
+            install_deps_ubuntu
+            fetch_and_unpack_fmtlib        
             ;;
         Darwin)
             install_deps_darwin

--- a/scripts/install-deps.sh
+++ b/scripts/install-deps.sh
@@ -437,7 +437,7 @@ main()
             ;;
         debian)
             install_deps_ubuntu
-            fetch_and_unpack_fmtlib        
+            fetch_and_unpack_fmtlib
             ;;
         Darwin)
             install_deps_darwin

